### PR TITLE
Add a note about adding Tools to ConversationalAgent

### PR DESCRIPTION
**Pull request by bilgeyucel**
Fixes https://github.com/deepset-ai/devrel-board/issues/113
* Briefly explain how people can use Tools with ConversationalAgent

### CODE REVIEW

File: tutorials/24_Building_Chat_App.ipynb [hunk 1]
```diff
@@ -1,1967 +1,442 @@
...
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "MiGGpDW98XjM"
+      },
+      "source": [
+        "### 4) Create a ConversationalAgent\n",
+        "\n",
+        "Now that we have all the necessary components, we can initialize the ConversationalAgent. When no tools are provided, the ConversationalAgent uses `conversational-agent-without-tools` prompt by default."
```
[the remark this answers is not in the document]
Comment (dfokina) [reply tutorials/24_Building_Chat_App.ipynb:213]: Changed it to an active voice here.

File: tutorials/24_Building_Chat_App.ipynb [hunk 2]
```diff
@@ -1,1967 +1,442 @@
...
+      "source": [
+        "> You can add tools to your chat application using `tools` params of the ConversationalAgent:\n",
+        "> ```python\n",
+        "> conversational_agent = ConversationalAgent(\n",
+        ">    prompt_node=prompt_node,\n",
+        ">    memory=summary_memory,\n",
+        ">    tools=[search_tool]\n",
+        ">)\n",
+        ">```\n",
+        ">To learn about how to create tools, check out [Haystack documentation](https://docs.haystack.deepset.ai/docs/agent#tools)."
```
Comment (dfokina) [inline tutorials/24_Building_Chat_App.ipynb:244]:
```suggestion
        ">To learn how to create tools, check out [Haystack documentation](https://docs.haystack.deepset.ai/docs/agent#tools)."
```

File: tutorials/24_Building_Chat_App.ipynb [hunk 3]
```diff
@@ -1,1967 +1,442 @@
...
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "RrPgO_1vzWb6"
+      },
+      "source": [
+        "\n",
+        "### Enabling Telemetry\n",
+        "Knowing you're using this tutorial helps us decide where to invest our efforts to build a better product but you can always opt out by commenting the following line. See [Telemetry](https://docs.haystack.deepset.ai/docs/telemetry) for more details."
```
Comment (dfokina) [inline tutorials/24_Building_Chat_App.ipynb:81]:
```suggestion
        "Knowing you're using this tutorial helps us decide where to invest our efforts to build a better product, but you can always opt out by commenting the following line. See [Telemetry](https://docs.haystack.deepset.ai/docs/telemetry) for more details."
```

Comment (dfokina) [reply tutorials/24_Building_Chat_App.ipynb:81]: This is the last one, otherwise looks great! :)

**Review by dfokina — APPROVED** [review]

**Review by TuanaCelik — APPROVED** [review]

**Pull request merged**